index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/styles.css">
+    <title>Fabricio La Torre</title>
+</head>
+<body>
+    <header class="main-header">
+        <div class="container">
+            <h1 class="mh-logo">
+              <a href="#">La Torre</a>
+                
+            </h1>
+            <!--
+            <nav class="main-nav">
+               
+                <ul class="main-nav-list">
+                  
+                    <li>
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                    <li>
+                        <a href="#"><img alt="maillogo" src="images/mail.svg"></a>
+                    </li>
+                </ul>
+            </nav>
+          -->
+              <h1 class="button">
+                  <a href="#"><img alt="hambutton" src="images/hambutton.svg"></a>
+              </h1>
+        </div>
+
+    </header>
+
+
+    <section class="body-container">
+   
+
+      <ul class="intro">
+        <li>
+          <h1 class="name">
+              Hey there! I'm Fabricio La Torre.
+          </h1>
+          <h3>
+              I'm a Web Developer 
+          </h3>
+          <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nobis doloremque ea sequi ullam dolor animi, cumque ab voluptate iusto sit.
+          </p>
```

Comment:
![paragraph](https://user-images.githubusercontent.com/93039359/177864182-dacf8d4e-6f75-4433-913b-2b6388f9fc3e.png)

- [ ] Instead of Lorem ipsum please add the paragraph shown in the Figma template :+1: Because your portfolio is showing you so you should add this paragraph to say what you can do :muscle: